src/applications/ivc-champva/10-10D/config/constants.js

```diff
@@ -31,7 +31,12 @@ export const REQUIRED_FILES = {
   applicantMedicarePartDCard: 'Medicare Card (Part D)',
   applicantMedicareIneligibleProof: 'Proof of Medicare Ineligibility',
   applicantOhiCard: 'Other Health Insurance Cards',
-  applicantOtherInsuranceCertification: 'VA Form 10-7959C',
+  applicantOtherInsuranceCertification: {
+    name:
+      'Completed and signed CHAMPVA Other Health Insurance (OHI) Certification ',
+    linkText: 'Get VA Form 10-7959c to download (opens in a new tab)',
+    href: 'https://www.va.gov/find-forms/about-form-10-7959c/',
+  },
```

Comment:
With the refactored `MissingFileList` component, we can now consume `REQUIRED_FILES` items that have associated links to display.